### PR TITLE
linuxPackages.r8168: 8.052.01 -> 8.053.00, remove broken

### PR DESCRIPTION
**Pull request by fairct**
## Description of changes
Bumping to latest r8168 module from mirror. Removed maintainer [per their request](https://github.com/NixOS/nixpkgs/issues/256329#issuecomment-2095392065) and updated broken to match [module's current compatibility](https://www.realtek.com/Download/List?cate_id=584) with kernels up to 6.8. 

Successfully built this module (without my changes) and applied it to an install (using allow broken) and it worked well. I'm not confident in the syntax I used for the `broken` setting, nor was I able to build this change locally (almost certainly because I don't know clearly what I'm doing - I tried `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD -p r8168"` for example and it hung my terminal for a while then subsequently crashed)

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/os-specific/linux/r8168/default.nix [hunk 1]
```diff
@@ -53,7 +53,7 @@ in stdenv.mkDerivation rec {
     '';
     license = licenses.gpl2Plus;
     platforms = platforms.linux;
-    maintainers = with maintainers; [ timokau ];
-    broken = (lib.versions.majorMinor kernel.modDirVersion) != "5.15";
+    maintainers = with maintainers; [];
+    broken = (lib.versions.majorMinor kernel.modDirVersion) > "6.8";
```
Comment (wegank) [inline pkgs/os-specific/linux/r8168/default.nix:57]:
```suggestion
    broken = lib.versionAtLeast kernel.modDirVersion "6.9";
```

Comment (fairct) [reply pkgs/os-specific/linux/r8168/default.nix:57]: can do - saw a reference to this syntax on an old PR but couldn't find a modern example - wasn't sure if the syntax was still valid

**Comment by fairct** [comment]
@timokau would you mind reviewing? not sure if I'm missing something or if there's more I can do to help move this along.

**Review by timokau — CHANGES_REQUESTED** [review]
Thanks for the update, and also thanks for removing my outdated maintainer entry :)

You wrote that you verified that the github version still matches the official one, which source did you use for the official one? Please update the link in the comment, the current one seems broken.

**Comment by fairct** [comment]
thanks timokau. after following the download link(s) through realtek I diff'd with the `mtorromeo` repo:
```
> diff -r r8168/ r8168-8.053.00
Only in r8168/: .git
```

**Review by timokau — APPROVED** [review]
Thanks! Sorry for the delay.

**Pull request merged**